src/Enum.php

```diff
@@ -188,6 +188,13 @@ public static function toArray(): array
         return array_combine(static::getValues(), static::getIndices());
     }
 
+    public static function getAll(): array
```

Comment:
The method should be added to the interface.

Reply:
should be done in the following `v2` PR

Reply:
Done

Reply:
https://github.com/spatie/enum/pull/33/commits/224cb21a0cb21c16f95ee7b17021846327019427